### ISSUE TITLE
Tell parent component if at least one fieldGroup field has an error

### ISSUE THREAD

**Issue by Steb95** [ngx-formly/ngx-formly#3235]
**Question**
Hi, I'm stuck with the following problem: I've this simple JSON object
<pre>{
    title: 'Test schema',
    type: 'object',
    properties: {
        testArray: {
            title: 'Test array',
            type: 'array',
            items: {
                title: 'Array item',
                type: 'object',
                required: ['firstInput'],
                properties: {
                    firstInput: {
                        type: 'input',
                    },
                    secondInput: {
                        type: 'integer',
                        minimum: 10
                    }
                }
            }
        }
    }
}</pre>
from which I create the fields object using `formlyJsonschema.toFieldConfig()`. I copied the object.type and array.type from https://formly.dev/examples/advanced/json-schema.

What I'd like to do is to add a custom css class to the array item, that is this part of array.type code:
<pre>
&lt;div *ngFor="let field of field.fieldGroup;let i = index;" class="row align-items-center">
	&lt;formly-field class="col" [field]="field">&lt;/formly-field>
	&lt;div *ngIf="field.templateOptions.removable !== false" class="col-2 text-right">
		&lt;button class="btn btn-danger" type="button" (click)="remove(i)">-&lt;/button>
	&lt;/div>
&lt;/div>
</pre>
if **any of that item child field** has an error, for example (for item n. 1) if "firstInput" is not filled or "secondInput" has a value < 10.

In short, is there a way to tell to the parent component (object.type or array.type) if at least one of the field in `fieldGroup` array has an error **whitout changing the original JSON schema**?
